src/renderer/store/modules/session.js

```diff
@@ -145,7 +145,7 @@ export default {
       state.avatar = 'pages/new-profile-avatar.svg'
       state.background = null
       state.currency = MARKET.defaultCurrency
-      state.timeFormat = 'Default'
+      state.timeFormat = 'default'
```

Comment:
In other places was written capitalized:

```suggestion
      state.timeFormat = 'Default'
```

Reply:
@luciorubeens thanks